README.md

```diff
@@ -27,12 +27,42 @@ From this SQLDelight will generate a `Database` Kotlin class with an associated
 
 #### Android
 ```groovy
+android {
+  ...
+  sourceSets {
+    main.java.srcDir("src/main/sqldelight")
+  }
+}
+
 dependencies {
+  ...
   implementation "com.squareup.sqldelight:android-driver:1.2.2"
 }
 ```
 ```kotlin
 val driver: SqlDriver = AndroidSqliteDriver(Database.Schema, context, "test.db")
+val database = Database(driver)
+```
+
+##### Android Studio
+
+[Follow these steps to get started](docs/android_studio_getting_started.md) if you are using Android Studio.
+
+##### Testing (Advanced)
```

Comment:
yup i like this. I think we're thinking of splitting out android into its own series of readmes and i think this should be included

@saket